content/blog/entries/science-museum-implementation/contents.lr

```diff
@@ -0,0 +1,115 @@
+title: Science Museum provider implementation
+---
+categories: 
+
+cc-catalog 
+gsoc
+gsoc-2020
+---
+author: srinidhi
+---
+series: gsoc-2020-cccatalog
+---
+pub_date: 2020-05-29
+---
+body:
+Science museum is a provider with around 80,000 CC license images, currently Science museum data is ingested from Common Crawl. 
```

Comment:
The reader won't necessarily know what 'Common Crawl' is.  Please either define it, or quickly introduce it and provide a link.